cmd/version.go

```diff
@@ -20,11 +26,60 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I'm easy, I think consistency across the codebase is more important, and so far how you have it here has been the convention.